### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Cena-JM**

### CODE REVIEW

File: src/controller.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const processWeatherdesc = (condition) => {
```
Comment (dipto0321) [inline src/controller.js:1]: **Optional tips:** Switch is the best choice for handling this operation. If you want then you can implement switch.

**Review by dipto0321 — CHANGES_REQUESTED** [review]
Hi @Cena-JM ,

Good job 👏 . This project is good. Here is the checklist for the application.

#### Application
- [x] It allows the user to search for a specific location
- [ ] It has a toggle control to determine if the result will be shown in Celsius or Fahrenheit
- [x] It changes its appearance based on the received data
- [x] It uses either `promises` or `async/await`

Please add that unchecked feature and send another request to re-review this project.

Thanks 🙏

**Comment by dipto0321** [comment]
Hi @Cena-JM ,

Good job 👏 . I liked the feature which you added. Don't remove this feature. Just add the actual feature with it as I discussed with you on slack and send another request to final review this project.

Thanks 🙏

**Comment by rvvergara** [comment]
Hi @Cena-JM and @dipto0321,

The last item in the checklist has already been satisfied (toggle button that converts between C and F). So I guess it's a 

# :+1: 

I'm not aware of the subject of the Slack conversation so if @dipto0321 has discussed something else that you need, @Cena-JM, then his veto will overrule this thumbs up.. But unless and until that happens this stands.

**Pull request merged**